docs/api/commands/getalllocalstorage.mdx

```diff
@@ -56,15 +56,15 @@ set to `value2` on `https://other.com`, `cy.getAllLocalStorage()` will yield:
 ### Get all localStorage
 
 ```javascript
-cy.visit('https://example.com', {
+cy.visit('/users', {
   onBeforeLoad(win) {
     win.localStorage.setItem('key', 'value')
   },
 })
 
 cy.getAllLocalStorage().then((result) => {
   expect(result).to.deep.equal({
-    'https://example.com': {
+    '/users': {
```

Comment:
```suggestion
    'http://localhost:8080': {
```

The key is the origin the local storage is associated with. Might also want to update above to `cy.visit('http://localhost:8080', {` as well.